src/orders/views/OrderDetails/OrderDraftDetails/index.tsx

```diff
@@ -114,7 +107,7 @@ export const OrderDraftDetails: React.FC<OrderDraftDetailsProps> = ({
     variables: {
       id: order?.user?.id
     },
-    skip: !order?.user?.id || !isAnyAddressEditModalOpen(params.action)
```

Comment:
Why is it deleted? Do we need addresses on draft view, when modal is not opened?

Reply:
Yeah, you're right, after some digging I realised it was unnecessary